CONTRIBUTING.rst

```diff
@@ -1,16 +1,5 @@
-If you would like to contribute to the development of OpenStack,
-you must follow the steps in this page:
+If you would like to contribute to the development of this project.
 
-   http://docs.openstack.org/infra/manual/developers.html
+Submit your pull request and issues to https://github.com/uggla/python-redfish .
```

Comment:
Issues should only be in one place under https://github.com/bcornec/python-redfish
For code, we should mention both repo I think showing yours as a clone more in advance of mine.